src/Microsoft.AspNet.Razor/RazorResources.resx

```diff
@@ -422,4 +422,7 @@ Instead, wrap the contents of the block in "{{}}":
     <value>Inline markup blocks (e.g. @&lt;p&gt;content&lt;/p&gt;) must not appear in non-string tag helper attribute values.
  Expected a '{0}' attribute value, not a string.</value>
   </data>
+  <data name="RewriterError_EmptyTagHelperBoundAttribute" xml:space="preserve">
+    <value>Attribute '{0}' on tag helper element '{1}' is a '{2}' tag helper bound attribute with no value. Only '{3}' typed attributes can be without a value.</value>
```

Comment:
also I'm not sure "tag helper bound attribute" is going to be meaningful to Razor authors /cc @DamianEdwards ?